src/components/NewCommentForm.tsx

```diff
@@ -1,37 +1,154 @@
-import React from 'react';
+import React, { useCallback, useContext, useState } from 'react';
+import { UserListContext } from './listContext';
+import { addComments } from './todosApi/api-todos';
 
 export const NewCommentForm: React.FC = () => {
+  const [autorNameInput, setAutorNameInput] = useState('');
+  const [emailInput, setEmailInput] = useState('');
+  const [commentTextInput, setCommentTextInput] = useState('');
+  const [autorNameError, setAutorNameError] = useState(false);
+  const [emailError, setEmailError] = useState(false);
+  const [bodyError, setBodyError] = useState(false);
```

Comment:
In case it is a boolean variable name should start from `is` or `has`, etc - https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig